src/components/notification/AposNotification.vue

```diff
@@ -111,9 +149,38 @@ export default {
     height: 12px;
   }
 
-  .apos-notification__label {
+  .apos-notification__label,
+  .apos-notification__progress-value {
     font-size: map-get($font-sizes, default);
+  }
+
+  .apos-notification__label {
     letter-spacing: 0.75px;
     color: var(--a-text-primary);
   }
+
+  .apos-notification__progress {
+    display: flex;
+    align-items: center;
+    flex-shrink: 0;
+  }
+
+  .apos-notification__progress-bar {
+    width: 70px;
+    height: 4px;
+    margin-left: 20px;
+    // BG color consistent between themes.
+    background-color: #2c354d;
+    border-width: 0;
+
+  }
+  .apos-notification__progress-now {
```

Comment:
set a transition property for width

Reply:
Done. You'll have to test it using dev tools. Since the knobs reloads the component you don't get the transition.